src/dmd/mars.d

```diff
@@ -1697,6 +1697,10 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
             else if (p[4])
                 goto Lerror;
         }
+        else if (arg == "-ctfe-cov")
```

Comment:
Maybe extend `-cov` to accept an optional argument instead of adding a new flag?
E.g. `-cov=run|with-ctfe`

Reply:
Nevermind

Reply:
Where would the numeric flag go? 
You'd have to use cov twice? `-cov=100 -cov=with-ctfe` ?

Reply:
In case you didn't know `-cov=` already accepts a value which is the desired coverage percentage.